config/locales/controllers/fr.yml

```diff
@@ -1,123 +1,122 @@
 ---
 fr:
   api_secrets_controller:
-    generated: 'Your API Key has been generated: %{secret}'
-    revoked: Your API Key has been revoked.
+    generated: 'Votre clé d API a été générée : %{secret}'
+    revoked: Votre clé API a été révoquée.
   application_controller:
-    bad_request: 'Error: Bad Request'
-    not_authorized: 'Error: not authorized'
-    please_sign_in: Please sign in
+    bad_request: 'Erreur : mauvaise demande'
+    not_authorized: 'Erreur : non autorisé'
+    please_sign_in : Veuillez vous connecter
   articles_controller:
-    deleted: Article was successfully deleted.
+    deleted: L'article a été supprimé avec succès.
   comments_controller:
     create:
-      success: created
-      failure: comment already exists
-      authorization_error: Your privileges have been suspended.
```

Comment:
This key is missing. That's why the spec was failing.

Reply:
Thanks @maestromac, I added `authorization_error` 👍🏼